src/Common/Numerals.fs

```diff
@@ -0,0 +1,8 @@
+module Common.Numerals
+
+type Range = 
```

Comment:
On the ranges themselves... I would suggest 0-20, 0-100, 0-1000 and 0+, that looks a little more natural. And things become kind of boring after 1000, you just need to learn words for big numbers, everything else is quite regular.